src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -301,7 +312,7 @@ docker run -p 9000:9500 \
 
 *Configuration*
```

Comment:
* -> section heading
* -> action title

Reply:
Changed to "Configuring properties"

Reply:
How about "Using environment variables to configure Che properties"?